doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
Could you run the notebook on your computer once, and keep the outputs? So they will show in the documentation.